fixtures/ios_app_with_framework_and_resources/App/Sources/AppDelegate.swift

```diff
@@ -19,10 +20,11 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         print(hello())
         print("AppDelegate -> \(framework1.hello())")
         print("Main bundle image: \(String(describing: UIImage(named: "tuist")))")
-        print("StaticFrameworkResouce image: \(String(describing: staticFrameworkResources.tuist))")
-        print("StaticFramework2Resouce image: \(String(describing: staticFramework2Resources.loadImage()))")
-        print("StaticFramework3Resouce image: \(String(describing: resourcesStaticFramework3.loadImage()))")
-        print("StaticFramework4Resouce image: \(String(describing: resourcesStaticFramework4.loadImage()))")
+        print("StaticFrameworkResource image: \(String(describing: staticFrameworkResources.tuist))")
+        print("StaticFramework2Resource image: \(String(describing: staticFramework2Resources.loadImage()))")
+        print("StaticFramework3Resource image: \(String(describing: resourcesStaticFramework3.loadImage()))")
+        print("StaticFramework4Resource image: \(String(describing: resourcesStaticFramework4.loadImage()))")
+        print("StaticFramework5Resource image: \(String(describing: UIImage(named:"StaticFramework5Resources-tuist", in: StaticFramework5Resources.bundle, compatibleWith: nil)))")
```

Comment:
minor: For this fixture to help represent the issue in https://github.com/tuist/tuist/issues/1602, `StaticFramework5` would need an asset catalog with the image within it.

Reply:
Hi @kwridan, I did the change, but I've refreshed my branch with `master` and now, the project doesn't compile :/ Is there any important change or something that I should do?

Reply:
Forget my comment, it's solved 👍 
I've added an asset catalog with the image inside

Reply:
ouch, what error are you seeing?

Reply:
no problem, it was a problem related to my IDE :) 
I've updated the PR with the changes 👍